apps/judicial-system/web/src/routes/DetentionRequests/DetentionRequests.tsx

```diff
@@ -32,6 +32,7 @@ export const DetentionRequests: React.FC = () => {
 
   useEffect(() => {
     document.title = 'Allar kröfur - Réttarvörslugátt'
+    window.localStorage.clear()
```

Comment:
looks good, but I wonder how this will affect the browser's back behaviour

I'll test it and report a separate issue if I find a problem